zcash_primitives/src/consensus.rs

```diff
@@ -139,6 +149,7 @@ impl TryFrom<u32> for BranchId {
             0x76b8_09bb => Ok(BranchId::Sapling),
             0x2bb4_0e60 => Ok(BranchId::Blossom),
             0xf5b9_230b => Ok(BranchId::Heartwood),
+            0x1111_1111 => Ok(BranchId::Canopy), //FIXME
```

Comment:
Canopy has a consensus branch ID now, but let's leave these as placeholders so that anything attached to the Canopy NU in this branch is explicitly domain-separated from anything that might end up on mainnet.